script/check-diff

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+# Make sure we notice any untracked files generated by the build
+git add --intent-to-add .
```

Comment:
This creeps outside the scope of the change you're introducing, but is related:

What do you think about replacing `--quiet` with `--exit-code`? My thinking is that we'd still trigger a build failure if the contents are different, but we'd be able to inspect the diff.

Reply:
That's a good call, I've modified the script to use `--quiet`, but in the event of a failure, output the full diff on line 8 (not in diff)